scripts/youtube_dataset_creation.sh

```diff
@@ -6,35 +6,35 @@ THREAD=$2
 export CUDA_VISIBLE_DEVICES=$CUDA_DEVICE
 
 YOUTUBE='https://www.youtube.com/channel/UCb9_Bhv37NXN1m8Bmrm9x9w'
-CHECKPOINT=best_checkpoints/JasperNetBig_NovoGrad_lr1e-4_wd1e-3_bs1024____long-train_bs1024_step6_430k_checkpoint_epoch247_iter0446576.pt
```

Comment:
given that `dataset_tools` is in better shape now, please move this this file there

Reply:
done